packages/cli/lib/constants.js

```diff
@@ -40,4 +42,4 @@ exports.FALLBACK_TEMPLATE_OPTIONS = [
 	},
 ];
 
-exports.TEMPLATES_CACHE_FILENAME = '.preact-templates.json';
+exports.TEMPLATES_CACHE_FILENAME = `${os.homedir()}/.cache/preact-templates.json`;
```

Comment:
use `path.join`. also, not sure if `.cache` exist in all OS.

Reply:
Yeah have to create if it doesn't

Reply:
done